pkg/cluster/cluster.go

```diff
@@ -1136,6 +1163,7 @@ func (c *Cluster) Switchover(curMaster *v1.Pod, candidate spec.NamespacedName) e
 	// close the label waiting channel no sooner than the waiting goroutine terminates.
 	close(podLabelErr)
 
+	c.eventRecorder.Eventf(c.GetReference(), v1.EventTypeNormal, "Switchover", "Switchover from %q to %q FAILED: %v", curMaster.Name, candidate, err)
```

Comment:
It seems this should be wrapped in `if err != nil {}` 
`FAILED: <nil>` is _technically_ correct, but unexpected when the switchover was successful

Reply:
Opened issue #1134